### ISSUE TITLE
Put a warning in the docs that individual developers will get rejected from publishing

### ISSUE THREAD

**Issue by thatusernamenotavailable** [OfficeDev/generator-office#538]
The only way to actually `publish` an Add-in using this repo is to be part of a larger company. Individuals and sole proprietors will get rejected from opening up a developer account at `https://partner.microsoft.com` even after submitting endless documentation. I feel like the README should warn individual developers BEFORE they start developing. 

![verification](https://user-images.githubusercontent.com/66221456/84820334-b4d5e400-afd6-11ea-98fe-3714136a1c45.png)

Please see: https://stackoverflow.com/questions/62416240/how-do-i-verify-a-company-account-so-i-can-publish-an-office-javascript-add-in

**Comment by pl4nty** [comment]
Agreed. Pretty disappointing after expecting AppSource to match every other extension marketplace (MS/Edge/Chrome etc), especially as paid apps aren't allowed.

**Comment by TCourtneyOwen** [comment]
@Rick-Kirkham @nokafor Can we move this issue to a more suitable place.  This is not a generator-office issue
